.github/CHANGELOG.md

```diff
@@ -33,6 +34,9 @@
 * The `simulations.py` module has been made simpler and more general. [#57](https://github.com/XanaduAI/flamingpy/pull/57) 
   * Functions within this module accept `code, noise, decoder` objects, as well as arguments (`code_args, noise_args, decoder_args`) separately.
 * Examples, benchmarks, and tests have been modified to take advantage of the simpler approach to noise application and the new `CVMacroLayer`. [#57](https://github.com/XanaduAI/flamingpy/pull/57) 
+* In `simulations.py`: [#93](https://github.com/XanaduAI/flamingpy/pull/93)
+  * Directory argument `-d` was removed as it was unused. A working version can be added in the future.
+  * We have removed reporting "decoding_total_time" as a more sophisticated time profiler feature systematically timing all the script subroutines will be added soon. To follow the progress of this new feature see [FTSTACK#91](https://github.com/XanaduAI/ft-stack/pull/91). 
```

Comment:
Community users might not be able to see this PR since FT-stack is a private repo